internal/build/phases.go

```diff
@@ -173,6 +186,7 @@ func (l *Lifecycle) newAnalyze(repoName, cacheName, networkMode string, publish,
 		return phaseFactory.New(configProvider), nil
 	}
 
+	// TODO: when platform API 0.2 is no longer supported we can delete this code: https://github.com/buildpacks/pack/issues/629.
```

Comment:
I'm not sure if it's really necessary but maybe you can indicate in the issue you opened (629) that part of the work is mentioned in some TODOs that contains the issue number in the code. In this way, whoever works on issue 629 will make sure to delete all these TODOs.